client/src/actions/tagDirectory.js

```diff
@@ -0,0 +1,35 @@
+import { FETCH_TAGS_PENDING, FETCH_TAGS_SUCCESS, FETCH_TAGS_ERROR } from '../constants';
```

Comment:
Is it a good idea to reuse the constants? doesn't this mean that the reducer will affect the other places we use these constants?

Reply:
They are not being re-used. I added them to `constants.js` down below.

Reply:
I can make them a little more specific to the Tag Directory Page though if you think that's needed.

Reply:
Ah my bad. Maybe they should be more specific but it's not super important